optimizely/decision_service.py

```diff
@@ -224,9 +225,7 @@ def get_stored_variation(self, project_config, experiment, user_profile):
 
         return None
 
-    def get_variation(
-            self, project_config, experiment, user_context, ignore_user_profile=False
-    ):
+    def get_variation(self, project_config, experiment, user_context, options=None):
```

Comment:
options were added by me and Jae because of special case of ignore user profile service. 
Python has ignore_user_profile argument that other sdk-s don't (is my understanding). That interfered with options. So we added ignore UPS into the get_variation function becasue it only appears there. Then we can freely use options as teh parameter. 

before it was a mix of ignore ups and options.
get_variation method above

Reply:
See this comment from Jae:

"Some functions expect decide "options" as a paramter. It's an array of decide option. But some of them (get_varaition_for_feature) accept "ignore_user_profile", a boolean value. You pass down boolean to array below.
We need to make them all consistent. I suggest to change them all to "options" and determine "ignore_user_profile" boolean value inside this function."